Sources/Layout/MessageSizeCalculator.swift

```diff
@@ -97,7 +97,8 @@ open class MessageSizeCalculator: CellSizeCalculator {
         let avatarVerticalPosition = avatarPosition(for: message).vertical
         let accessoryViewHeight = accessoryViewSize(for: message).height
         let accessoryViewVerticalPadding = accessoryViewPadding(for: message).vertical
-        let accessoryViewTotalHeight = accessoryViewHeight + accessoryViewVerticalPadding
+        let accessoryViewYOriginInset = cellTopLabelHeight + messageTopLabelHeight + messageContainerPadding(for: message).top
+        let accessoryViewTotalHeight = accessoryViewHeight + accessoryViewVerticalPadding + accessoryViewYOriginInset
```

Comment:
This should just be:
```swift
let accessoryViewTotalHeight = accessoryViewHeight + accessoryViewVerticalPadding
```
No need for `accessoryViewYOriginInset`, see the reason why below

Reply:
The reason is because the `cellContentHeight` logic was already fine as is 😆. Do you mind cleaning up the git history by squashing these commits? Since we flip flopped back and forth

Reply:
Reverted